modules/database/src/std/dev/devSSiSoft.cpp

```diff
@@ -0,0 +1,91 @@
+/*************************************************************************\
+* Copyright (c) 2020 Michael Davidsaver
+* EPICS BASE is distributed subject to a Software License Agreement found
+* in file LICENSE that is included with this distribution.
+\*************************************************************************/
+
+#ifndef USE_TYPED_DSET
+#  define USE_TYPED_DSET
+#endif
```

Comment:
You don't need to do this at all, the modules/database/src/std/Makefile sets both RSET and DSET flags for all Base device support and record types.